src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -53,101 +54,35 @@
 /**
  * Controller responsible for "Sell > Catalog > Attributes & Features > Features" page
  */
-class FeatureController extends FrameworkBundleAdminController implements LegacyControllerBridgeInterface, LegacyListControllerBridgeInterface
+class FeatureController extends FrameworkBundleAdminController implements FrameworkBridgeControllerInterface
 {
-    use AdminControllerTrait;
-    use SmartyTrait;
-
-    /**
-     * This parameter is needed by legacy hook, so we can't remove it.
-     *
-     * @var string
-     */
-    public $php_self;
-
-    /**
-     * This parameter is needed by legacy helper shop, so we can't remove it.
-     *
-     * @var bool
-     */
-    public $multishop_context_group = true;
-
-    /**
-     * This parameter is needed by legacy helper shop, we can't remove it.
-     *
-     * @var int
-     */
-    public $multishop_context;
-
-    /**
-     * @var ControllerConfiguration
-     */
-    public $controllerConfiguration;
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getTable(): string
-    {
-        return 'feature';
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getClassName(): string
-    {
-        return 'Feature';
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getIdentifier(): string
-    {
-        return 'id_feature';
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getPositionIdentifier(): string
-    {
-        return 'id_feature';
-    }
+    use FrameworkBridgeControllerTrait;
+    use FrameworkBridgeControllerListTrait;
+    use FrameworkControllerSmartyTrait;
```

Comment:
When I see the diff between this and the previous code I think the refacto was worth it It feels much cleaner now ^^ We have much less noise